### DISCUSSION TITLE
Can't get authorization code

### DISCUSSION THREAD

**Discussion by youngshook** [spencerwooo/onedrive-vercel-index#334] [Q&A]
**A1 Account** 

**Step 2/3: Get authorisation code:**

Redirected auth URL:

http://localhost/?error=invalid_client&error_description=AADSTS650051%3a+Using+application+%27shxxxdrive%27+is+currently+not+supported+for+your+organization+xxxxx.edu.cn+because+it+is+in+an+unmanaged+state.+An+administrator+needs+to+claim+ownership+of+the+company+by+DNS+validation+of+xxxxxx.edu.cn+before+the+application+shxxxdrive+can+be+provisioned.%0d%0aTrace+ID%3a+3503fedd-c610-xxxx-964f-bae658693f00%0d%0aCorrelation+ID%3a+5d1f9bd4-ccd8-xxxx-bbd0-045e9f374946%0d%0aTimestamp%3a+2022-01-31+02%3a23%3a04Z#

Does this mean my A1 account is unavailable?

**Comment by spencerwooo** [comment]
I believe you can still use this if you provide your own clientId and clientSecret

Check out: https://onedrive-vercel-index.spencerwoo.com/docs/advanced#using-your-own-clientid-and-clientsecret
